coremltools/models/model.py

```diff
@@ -121,6 +123,9 @@ class NeuralNetworkShaper(object):
 
     def __init__(self, model, useInputAndOutputShapes=True):
 
+        msg = "Class {} will be deprecated from version 4"
+        warnings.warn(msg.format('NeuralNetworkShaper'), category=FutureWarning)
+
```

Comment:
Decorator should work on the class whenever an object is instantiated.

IMO, let's decorate the class with the same decorator.

Reply:
Done.